examples/whisper.py

```diff
@@ -229,12 +222,28 @@ def transcribe_file(model, enc, filename):
     if lst[-1] == enc._special_tokens["<|endoftext|>"]:
       return  transcription
 
+def init_whisper(model_name):
+  filename = BASE / "whisper-{}".format(MODELS[model_name].split('/')[-1])
+  download_file(MODELS[model_name], filename)
+  state = torch_load(filename)
+  model = Whisper(state['dims'])
+  load_state_dict(model, state['model_state_dict'])
+  enc = get_encoding('gpt2' if model_name.endswith(".en") else 'multilingual', state['dims']['n_vocab'])
+  return model, enc
 
 if __name__ == "__main__":
-  model, enc = init_whisper("small.en" if getenv("SMALL") else "tiny.en")
+  parser = argparse.ArgumentParser(formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+  parser.add_argument("--audio", default=None, type=str, help="audio file to transcribe")
+  parser.add_argument("--model", default="tiny.en", choices=MODELS.keys(), help="name of the Whisper model to use")
+
+  args = parser.parse_args().__dict__
```

Comment:
Why not just `args.model` and `args.audio`?

Reply:
good point, changed it now

Reply:
I still don't know why you need `__dict__`

Reply:
we don't need `__dict__` indeed, I had copied it from an example...removed it now